hal/src/main/native/systemcore/CAN.cpp

```diff
@@ -65,24 +66,30 @@ struct FrameStore {
 };
 
 struct SocketCanState {
-  wpi::EventLoopRunner loopRunner;
+  wpi::EventLoopRunner readLoopRunner;
+  wpi::EventLoopRunner writeLoopRunner;
```

Comment:
It's a little strange to have two loops.  Are there blocking calls happening?

Reply:
The writes technically can block. And we kind of want them to in order to no lose packets to the socketcan buffer. I could make it non blocking with another queue, but thats kind of a PITA. Willl rethink when adding more buses.

Reads we will definitely want on their own loop though.

Reply:
Digging through more, we will definitely want separate loops if we ever add ACK checking. The way socketcan does ACK checking is kind of interesting.